content/articles/missing-values-in-time-series/index.md

```diff
@@ -0,0 +1,230 @@
+### A Guide to Handle Time Series Missing Values
+When working with real-world data, it's a common challenge to encounter missing values. This article will guide us in addressing such a problem in time series data.
+
+### Prerequisites:
+- Have a good understanding of how to work with time series data in numpy
+- [Dataset](https://github.com/DennisKimt/datasets/commit/28bc7c1804279d8401b4dd399fe264852d719655)
+- Access to the Jupiter Notebook or Google Colab
+
+### Introduction
+Time series data normally differ from the machine learning datasets. This is because in time series, data is collected in different conditions over time, and various mechanisms may contribute to missing records in different periods in time. Those mechanisms are known as **Missingness Mechanisms**.
+
+There are three types of missing data.
+
+1. **Missing Completely at Random(MCAR):** In simple terms, *MCAR* means there is no relationship between the missing data and already observed data. The probability of the missing data is completely random and is not dependent on already observed data, i.e., $P(Missing \ | \ Complete \ data)=p(Missing)$.
+
+1. **Missing at Random(MAR):** A variable is missing at random if the probability of missingness depends only on the available information, i.e., $P(Missing \ | \ Complete \ data)=p(Missing \ | \ Observed \ data)$.
+1. **Missing not at Random(MNAR):** The probability of missingness, in this case, depends on the variable itself.
+
+Time series models work with complete data, and therefore they require the missing data to be replaced with meaningful values before actual analysis. At a high level, missing values in time series are handled in two ways, either dropping them or replacing them. However, dropping missing values can be an inappropriate solution due to the time order of the data and the correlation between observations in adjacent periods.
+
+Estimating a reasonable value such that the components of the series are not distorted is a great approach to dealing with missing values in time series. The process of replacing missing values with values estimated from the same data or observed from the environment with the same conditions underlying the missing data is called *Imputation*.
+
+In python, let's look at various *imputation techniques* used in time series.
+
+### Python Implementation
```

Comment:
Let's have the steps clearly highlighted and well explained. By that I mean, 
Step one: ....